platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -60,6 +60,13 @@ The "convention" concept, defined by the `org.gradle.api.plugins.Convention` typ
 
 Core Gradle plugins that still registered deprecated conventions have also been updated, leading to their removal:
 
+.Removed plugin conventions
+[cols="1,1,1,5"]
+|===
+|Plugin |Access |Type |Solution
+|`base` | `project.distDirName`, `project.libsDirName`, `project.archivesBaseName` | `BasePluginConvention` | Replaced by `project.base` extension of type `BasePluginExtension`.
+|===
+
 .Removed plugin extensions
```

Comment:
Arg, these two tables should be just one and titled `Removed plugin conventions` like you did. Using "extensions" in the existing table was a brain fart, it is only about "conventions".

Reply:
I can do that, no worries.

Reply:
Fixed